bcipy/display/paradigm/matrix/display.py

```diff
@@ -127,6 +128,21 @@ def _stim_positions(self) -> Dict[str, Tuple[float, float]]:
             for sym, stim in self.stim_registry.items()
         }
 
+    def capture_grid_screenshot(self, file_path: str) -> None:
+        """Capture Grid Screenshot.
+
+        Capture a screenshot of the current display and save it to the specified filename.
+        """
+        # draw the grid and flip the window
+        self.draw_grid(opacity=self.full_grid_opacity)
+        self.task_bar.current_index = 0
```

Comment:
I would't expect this method to mutate the object state. It seems like this has the potential to lead to some unexpected bugs. Maybe have a temp variable here and set it back to its previous value after the screenshot?